test/evergreen.yml

```diff
@@ -5486,6 +5486,7 @@ buildvariants:
   expansions:
     CMAKE_TOOLCHAIN_FILE: -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_stable_clang.cmake
     CMAKE_BUILD_TYPE: -DCMAKE_BUILD_TYPE=MSan
+    posix_configure_flags: -DENABLE_LZ4=0 -DENABLE_SNAPPY=0 -DENABLE_ZLIB=0 -DENABLE_ZSTD=0
```

Comment:
Will test/format know it cannot enable those libs?
Can we add a comment?
```
MemorySanitizer requires that all program code is instrumented, otherwise it may generate false reports.
```

Reply:
Yes, test format knows that.